views/index.hbs

```diff
@@ -1,2 +1,17 @@
 <h1>{{title}}</h1>
-<p>Welcome to {{title}}</p>
\ No newline at end of file
+<p>Welcome to {{title}}</p>
+
+{{!-- link to go to the create celebrity page
+ --}}<a href="celebrities/create">Create a celebrity!</a>
+
+{{!-- link to the celebrity list page
+ --}}<a href="celebrities">Celeb List</a>
+
+<a href="/movies/movies"> Movie List</a>
+
+<a href="/movies/new-movie">Create a new Movie!</a>
```

Comment:
Here what you are putting inside the `href` is the path to the `hbs` file in your folder structure.
But what you should put here is the `url` `path` that you specified in the `routes` file:

```
<a href="/celebrities/create">Create a celebrity!</a>
<a href="/celebrities">Celeb List</a>
<a href="/movies"> Movie List</a>
<a href="/movies/create">Create a new Movie!</a>
```